browser-core-docker/launcher

```diff
@@ -2,21 +2,28 @@
 
 # This script provides a simple interface for folks to use the docker install
 
-TAG=latest
+TAG=v0.18.1
```

Comment:
we shouldn't hard-code the version, right?

Reply:
Sorry that was left from testing. I'll patch it.